db/schema_0.sql

```diff
@@ -749,6 +749,22 @@ CREATE TABLE `application_stats` (
     ON DELETE CASCADE ON UPDATE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+DROP TABLE IF EXISTS `global_stats`;
```

Comment:
model on `application_stats` instead, so we can add global stats without needing a schema change.